### PR TITLE
WIP: Helm lint for Hono chart

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+
+  build-chart:
+    runs-on: ubuntu-latest
+    needs: check-for-chart-changes
```
Comment (sophokles73) [inline .github/workflows/ci.yaml]: does this mean that a *check-for-chart-changes* action is required to run first? If so, where is it defined?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Download packages helm chart
+        uses: actions/download-artifact@v1
+        with:
+          name: helm-chart
+      - run: mkdir -p charts
+      - run: cd charts && tar xzf ../eclipse-hono-chart.tar.gz --strip-components 2
+      - name: Run chart-testing (lint)
+        uses: helm/chart-testing-action@master
```
Comment (sophokles73) [inline .github/workflows/ci.yaml]: is this also a required action?

**Comment by ctron** [comment]
Ok, there is a missing feature in the helm lint action. That is why the lint command cannot be applied to the Hono chart. As we build the chart with maven first.

I will take a look at this next year. Please to not merge this PR, as it currently doesn't work and isn't complete.

**Comment by sophokles73** [comment]
@ctron now that we moved the Helm chart to the Eclipse IoT Packages project, I believe that this PR is obsolete, right?

**Comment by ctron** [comment]
This PR is indeed obsolete now.

**Pull request closed without merging**